data/processors/blank_columns.py

```diff
@@ -0,0 +1,24 @@
+import csv
+import sys
+import argparse
+
+# init arg parser
+parser = argparse.ArgumentParser()
+parser.add_argument('--entity', type=str, required=True)
+args = parser.parse_args()
+
+# init the incoming data as a dict reader
+lines = iter(line.decode('utf-8').strip() for line in sys.stdin.buffer.readlines())
+header = next(lines).split(',')
+reader = csv.DictReader(lines, fieldnames=header)
```

Comment:
Hm, can you not do:

```python
reader = csv.DictReader(sys.stdin)
```

[If fieldnames is omitted, the values in the first row of file f will be used as the fieldnames.](https://docs.python.org/3/library/csv.html#csv.DictReader)

Reply:
nice! much better.